plugins/github-deployments/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@backstage/plugin-github-deployments",
+  "version": "0.1.1",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "build": "backstage-cli plugin:build",
+    "start": "backstage-cli plugin:serve",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "diff": "backstage-cli plugin:diff",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/catalog-model": "^0.7.4",
+    "@backstage/core": "^0.7.3",
+    "@backstage/plugin-catalog-react": "^0.1.3",
+    "@backstage/theme": "^0.2.5",
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "4.0.0-alpha.45",
+    "@octokit/graphql": "^4.6.1",
+    "moment": "^2.29.1",
```

Comment:
https://backstage.io/docs/architecture-decisions/adrs-adr010

we really should not be pulling in `moment`.